charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -325,6 +390,83 @@ volumeMounts:
   - name: work
     mountPath: /actions-runner/_work
     {{- end }}
+    {{- if $mountGitHubServerTLS }}
+  - name: github-server-tls-cert
+    mountPath: {{ clean (print $tlsConfig.runnerMountPath "/" $tlsConfig.certificateFrom.configMapKeyRef.key) }}
+    subPath: {{ $tlsConfig.certificateFrom.configMapKeyRef.key }}
+    {{- end }}
   {{- end }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{- define "gha-runner-scale-set.default-mode-runner-containers" -}}
+{{- $tlsConfig := (default (dict) .Values.githubServerTLS) }}
+{{- range $i, $container := .Values.template.spec.containers -}}
+- name: {{ $container.name }}
+  {{- if eq $container.name "runner" -}}
```

Comment:
do we want to check `{{- if eq $container.name "runner" -}}` first and ignore other container?

ex:
```yaml
{{- if ne $container.name "runner" -}}
- {{ $container | toYaml | nindent 2 }}
{{- else }}
- name: {{ $container.name }}
  .....
{{- end }}
```